packages/components/tree-v2/src/composables/useTree.ts

```diff
@@ -0,0 +1,295 @@
+import { computed, nextTick, ref, shallowRef, watch } from 'vue'
+import {
+  NODE_CLICK,
+  NODE_COLLAPSE,
+  NODE_EXPAND,
+  CURRENT_CHANGE,
+  TreeOptionsEnum,
+} from '../virtual-tree'
+import { useCheck } from './useCheck'
+import { useFilter } from './useFilter'
+import type {
+  TreeProps,
+  TreeNodeData,
+  TreeKey,
+  TreeNode,
+  TreeData,
+  Tree,
+} from '../tree.type'
+
+export function useTree(props: TreeProps, emit) {
+  const expandedKeySet = ref<Set<TreeKey>>(new Set(props.defaultExpandedKeys))
+  const currentKey = ref<TreeKey | undefined>(undefined)
```

Comment:
`undefined` can be omitted.
```suggestion
  const currentKey = ref<TreeKey | undefined>()
```